src/main/java/com/zufar/onlinestore/payment/exception/PaymentEventParsingException.java

```diff
@@ -0,0 +1,14 @@
+package com.zufar.onlinestore.payment.exception;
+
+import lombok.Getter;
+
+@Getter
+public class PaymentEventParsingException extends RuntimeException {
+
+    private final String eventType;
+
+    public PaymentEventParsingException(String eventType) {
+        super(String.format("Payment event with '%s' cannot be parsed.", eventType));
```

Comment:
"PaymentEvent = '%s' cannot be parsed.", eventType)

Reply:
Done